opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/jms/JmsArtemisManualServerTest.java

```diff
@@ -106,15 +106,15 @@ public void stop() throws Exception {
     }
 
     public static class MsgListener {
-      private Message message;
+      private String message;
 
-      public Message getMessage() {
+      public String getMessage() {
         return message;
       }
 
       @JmsListener(destination = "fooQueue")
-      public void processMessage(Message msg) throws Exception {
-        log.info("Received msg: " + msg.toString());
+      public void processMessage(String msg) throws Exception {
```

Comment:
Shouldn't be tested also `Message`?

Reply:
We do in JmsStarter test.
But I'll try to add some more.

Reply:
quietly agree :P

Reply:
hi @alesj I would like to merge this. Are tests still on your TODO?

Reply:
@pavolloffay still a TODO, but no time atm unfortunately ...